tests/kernel/install_ltp.pm

```diff
@@ -224,12 +224,24 @@ sub add_ltp_repo {
     zypper_ar($repo, name => 'ltp_repo');
 }
 
-sub install_from_repo {
-    my $pkg = get_var('LTP_PKG', (want_stable && is_sle) ? 'qa_test_ltp' : 'ltp');
+sub get_default_pkg {
+    return (want_stable && is_sle) ? 'qa_test_ltp' : 'ltp';
+}
 
-    zypper_call("in --recommends $pkg");
-    script_run "rpm -qi $pkg | tee /opt/ltp_version";
-    assert_script_run "find " . get_ltproot() . q(/testcases/bin/openposix/conformance/interfaces/ -name '*.run-test' > ~/openposix-test-list);
+sub install_from_repo {
+    my $default_pkg      = get_default_pkg;
+    my @pkgs             = (get_var('LTP_PKG', $default_pkg));
```

Comment:
what about, so we can specify both packages in that variable
```suggestion
    my @pkgs             = split(/\s*,\s*/, get_var('LTP_PKG', $default_pkg));
```

I didn't checked other usage of `LTP_PKG`

Reply:
Well, this sounds better (I was thinking about it myself): instead of `LTP_PKG=both` which sounds confusing, just allow to define more packages: `LTP_PKG=ltp ltp-32bit`. @richiejp any opinion about this?